File: pgm/ABC_Visual_Basic/ABC_154_A.vb
Public Class Main
    Public Shared Sub Main(ByVal args() As String)
        Dim S, T, U as String
        Dim A, B as Integer
        Dim arr(2) as String
        arr = Split(Console.ReadLine())
        S = arr(0) : T = arr(1)
        arr = Split(Console.ReadLine())
        A = arr(0) : B = arr(1)
        U = Console.ReadLine()
        If S = U
            A -= 1
        Else
            B -= 1
        End If
        Console.WriteLine(A.ToString() & " " & B.ToString())
    End Sub
End Class

File: pgm/ABC_C#/ABC_154_A.cs
using System;

class Program
{
    static void Main()
    {
        String[] arr = Console.ReadLine().Split();
        string S = arr[0], T = arr[1];
        arr = Console.ReadLine().Split();
        int A = int.Parse(arr[0]), B = int.Parse(arr[1]);
        string U = Console.ReadLine();
        if (U == S) A--;
        else B--;
        Console.WriteLine(A + " " + B);
    }
}